### PR TITLE
Add CLI tool

### LINKED ISSUE #47: Simple CLI for common operations
It would be useful to have a `datapackage` binary where you could run commands like:

```
datapackage validate /path/to/datapackage.json
```

Some work was started in https://github.com/okfn/datapackage-validate-py/issues/7 and https://github.com/okfn/datapackage-validate-py/pull/8, but as we merged `datapackage-validate-py` here, I'm creating this issue.

### PR DESCRIPTION
**Pull request by scls19fr**
this fixes #47

### CODE REVIEW

File: datapackage/cli_datapackage_validate.py [hunk 1]
```diff
@@ -0,0 +1,45 @@
+from __future__ import print_function
```
Comment (pwalsh) [inline datapackage/cli_datapackage_validate.py]: I suggest housing this in `datapackage/cli.py`

File: datapackage/cli_datapackage_validate.py [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+
```
Comment (pwalsh) [inline datapackage/cli_datapackage_validate.py]: Other imports for Py2/3 consistency would be desirable: https://github.com/okfn/coding-standards#python-23

File: datapackage/cli_datapackage_validate.py [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+import click
+import os
+import sys
+import codecs
+import datapackage_validate
```
Comment (pwalsh) [inline datapackage/cli_datapackage_validate.py]: incorrect import

File: datapackage/cli_datapackage_validate.py [hunk 4]
```diff
@@ -0,0 +1,45 @@
+from __future__ import print_function
+
+import click
+import os
+import sys
+import codecs
```
Comment (pwalsh) [inline datapackage/cli_datapackage_validate.py]: I think `io.open` is a better choice for Py2/3 compat libs. See https://mail.python.org/pipermail/python-list/2015-March/687124.html

**Comment by scls19fr** [comment]
There is still some errors

````python
$ python datapackage/cli.py validate tests/schemas/data-package.json
Traceback (most recent call last):
  File "datapackage/cli.py", line 10, in <module>
    from datapackage import validate, exceptions
  File "/Users/scls/github/scls19fr/datapackage-py/datapackage/datapackage.py", line 15, in <module>
    import datapackage.schema
ImportError: No module named 'datapackage.schema'; 'datapackage' is not a package
```

**Comment by pwalsh** [comment]
It is because of an API change from the version you PR'd against on the other repo. To get validate, first create a `DataPackage` instance, and then call validate on it.

**Comment by scls19fr** [comment]
I think it's also a package / module problem. I let you or @vitorbaptista fix it.

**Comment by rgrp** [comment]
@scls19fr do you still want this to go in? Any chance you can fix the outstanding issue?

**Comment by scls19fr** [comment]
I'm quite busy these days.
Please go ahead @vitorbaptista or @pwalsh

**Comment by sirex** [comment]
I fixed this pull request here: https://github.com/scls19fr/datapackage-py/pull/1

**Comment by vitorbaptista** [comment]
Thanks for your work, @sirex. I'll take a look in the next few days :+1:

**Comment by roll** [comment]
Basic CLI is integrated to the lib starting from version `v1.0.0-alpha11` - https://github.com/frictionlessdata/datapackage-py#cli. So closing this PR in favor of implemented CLI bases on OKI standard `click` library.

**Pull request closed without merging**